VideoAPI/VideoApi.Contract/Responses/MeetingRoomResponse.cs

```diff
@@ -29,5 +29,10 @@ public class MeetingRoomResponse
         /// The pexip node to connect to for self test
         /// </summary>
         public string PexipSelfTestNode { get; set; }
+
+        /// <summary>
+        /// Public switched telephone network pin
+        /// </summary>
+        public string TelephoneConferenceId { get; set; }
```

Comment:
why is this string?

Reply:
Kinly return an 8 digit numeric value as a string

`"pstnPin":"62695113"`

Reply:
oops just saw this comment. kinly sending as a string, could send a pin starting with "0"?